package.json

```diff
@@ -25,7 +25,8 @@
     "prettier": "2.0.5",
     "typescript": "3.8.3",
     "mocha": "8.0.1",
-    "chai": "4.2.0"
+    "chai": "4.2.0",
```

Comment:
Is mocha/chai still being used elsewhere? In the future, we should think about having one test runner, either mocha or jest

Reply:
Yes, chai can be removed here 👍

Reply:
mocha is used for the LSP unit tests, but I'll create a separate PR for this